Backend/editor/tests/transform_tests.go

```diff
@@ -0,0 +1,136 @@
+package editor
```

Comment:
even though this says: "package editor" the package it's not really in the same package as your functions, this is the package: `cms.csesoc.unsw.edu.au/editor/tests/` while your functions are in `cms.csesoc.unsw.edu.au/editor` hence why you are getting import errors, you're trying to access private functions (functions that start with lower case letters) from an external package